### PR TITLE
[bug 1147227] Add notes about frames

### PR DESCRIPTION
**Pull request by willkg**
@mythmon Can you eyeball these comments? Is the language correct?

### CODE REVIEW

**Comment by mythmon** [comment]
Sounds good to me.

I think the Travis failure is related to the cache problem you fixed recently?

**Comment by willkg** [comment]
Yes. The travis failure is fixed in pull #536.

**Comment by mythmon** [comment]
Cool. r+

**Comment by willkg** [comment]
I'm going to change this around and instead of adding the comment to settings, I'll add a test to make sure the header is there for the feedback form.

**Comment by willkg** [comment]
^^^ That changes the notes a bit and also adds a test.

r?

File: fjord/feedback/tests/test_views.py [hunk 1]
```diff
@@ -26,6 +26,17 @@ def test_sad_redirect(self):
...
+    def test_xframe_header(self):
+        """Feedback form should *always* have X-FRAME-OPTIONS: DENY
+
+        This is because the feedback form can do various magic things
+        that shouldn't ever be framed.
+
+        """
+        url = reverse('feedback', args=(u'firefox',))
+        resp = self.client.get(url)
+        eq_(resp['X-Frame-Options'], 'DENY')
```
Comment (mythmon) [inline fjord/feedback/tests/test_views.py]: I'd also check for 200 OK, since your 40x/50x page probably has different X-Frame-Options properties.

**Comment by mythmon** [comment]
I'd be more comfortable if this was rebased onto master to pick up that cache config fix so Travis could run this test. But the test looks right. I'd suggest making it a little more defensive too though.

**Comment by willkg** [comment]
I can do that, but that hasn't been reviewed, yet.

**Comment by willkg** [comment]
^^^ Rebases against master with the CACHES fix. I'll keep an eye on Travis.

**Comment by mythmon** [comment]
Travis passed it. r+

**Comment by willkg** [comment]
7e796de [bug 1147227] Add note to feedback view and test re: X-FRAME-OPTIONS

Thank you!

**Pull request merged**